sdcm/cluster.py

```diff
@@ -1958,6 +1958,15 @@ def install_scylla_debuginfo(self):
             self.remoter.run(r'sudo apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated {0}-server-dbg={1}\*'
                              .format(self.scylla_pkg(), self.scylla_version), ignore_status=True)
 
+    def is_scylla_install(self):
```

Comment:
Ok, if you wanna go this way, there are few things to change in this PR:

`is_scylla_install` -> `is_scylla_installed`

Reply:
fixed